packages/backend/src/open_payments/payment/incoming/model.ts

```diff
@@ -245,7 +245,7 @@ export class IncomingPayment
     ilpStreamConnection: Connection
   }): OpenPaymentsIncomingPayment {
     return {
-      id: this.id,
+      id: this.url,
```

Comment:
(Copying comment from other [PR](https://github.com/interledger/rafiki/pull/998) for visibility):

> OpenPayment API IncomingPayments return the full url instead of just the uuid. This reflects what is done in the IncomingPaymentRoutes, but standardizes it when converting between IncomingPayment model and what Open Payments expects